app/src/main/java/com/example/parkingsystemkotlin/mvp/view/DialogDateResarvationView.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.parkingsystemkotlin.mvp.view
+
+import androidx.fragment.app.DialogFragment
+import com.example.parkingsystemkotlin.fragments.DateReservationDialogFragment
+import com.example.parkingsystemkotlin.mvp.contract.DialogDateResarvationContract
+import com.example.parkingsystemkotlin.mvp.view.base.FragmentView
+
+class DialogDateResarvationView(
+    fragment: DialogFragment,
+) : FragmentView(fragment), DialogDateResarvationContract.DialogFragmentDateReservationView {
+
+    override fun dismissPicker() {
+        (fragment as DateReservationDialogFragment?)?.dismiss()
```

Comment:
mmm in the FragmentView you not have something to invoke to the fragment?

Reply:
Done